models/plugin/server/manager.go

```diff
@@ -189,3 +204,50 @@ func (m *Manager) NewWorkConn(content *NewWorkConnContent) (*NewWorkConnContent,
 	}
 	return content, nil
 }
+
+func (m *Manager) NewUserConn(userConn *NewUserConn) {
```

Comment:
The logic of `NewUserConn` is same with `NewWorkConn`.

No need to use channel and cache.